spec/models/concerns/searchable_spec.rb

```diff
@@ -0,0 +1,7 @@
+describe Searchable do
+  describe ".rebuild_index" do
+    it "should rebuild the index" do
```

Comment:
should probably `create(:proposal)` here at least once so there is something to index.